src/shared/mount-util.c

```diff
@@ -1756,8 +1756,7 @@ int make_fsmount(
 
                 eq = strchr(word, '=');
                 if (eq) {
-                        *eq = 0;
-                        eq++;
+                        *eq++ = 0;
```

Comment:
This doesn't make the intention clearer. I'd drop the change.

Reply:
Fixed!